src/main/java/org/folio/consortia/utils/HelperUtils.java

```diff
@@ -0,0 +1,15 @@
+package org.folio.consortia.utils;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class HelperUtils {
+
+  private HelperUtils() {}
+
+  public static void isIdentical(String firstString, String secondString)
+  {
+    if(!StringUtils.equals(firstString, secondString)) {
+      throw new IllegalArgumentException("Arguments are not matching");
```

Comment:
It will be not clear for users what arguments are not matching, maybe we can add additional param to this method errorMessage.
Also I propose to rename it to be simialr with other namings: checkIdenticalOrThrow

Reply:
fixed